src/cript/nodes/util/__init__.py

```diff
@@ -37,6 +38,14 @@ def default(self, obj):
                 if uid in NodeEncoder.handled_ids:
                     return {"node": obj._json_attrs.node, "uid": uid}
 
+            try:
+                uuid_str = str(obj.uuid)
```

Comment:
obj is a bad variable name, can we name it something more descriptive?

Reply:
This is the variable name that the documentation of the `json` module uses.
We should not deviate from that.

Reply:
okie dokes, that makes sense